README.md

```diff
@@ -1,10 +1,10 @@
 # IBLT
 
-Invertible bloom lookup table implementation in rust.
+An Invertible Bloom Lookup Table implementation in the Rust programming language.
```

Comment:
I think `in XXX` is quite a common sentence.
(eg. [ed25519-dalek](https://github.com/dalek-cryptography/ed25519-dalek))